pkg/metrics/grafana/tidb_resource_control.json

```diff
@@ -3073,7 +3073,7 @@
               "format": "time_series",
               "interval": "",
               "intervalFactor": 2,
-              "legendFormat": "{{name}}-{{instance}}",
+              "legendFormat": "{{resource_group}}-{{instance}}",
```

Comment:
pd already changed in https://github.com/tikv/pd/pull/7547. maybe adjust it together.

Reply:
+1

Reply:
If we change other already used expressions after the user upgrades from the old version, old data will be lost due to expression change. We can change background expressions because it is an experimental feature.

Reply:
got